### PR TITLE
User dashboard

### PR DESCRIPTION
**Pull request by jamigibbs**
User Dashboard frontend component work. Also integrated neo4j and started working out the routes.

### CODE REVIEW

File: client/store/pathReducer.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import axios from 'axios'
 import { pathsDemo } from './path_demo_data'
 import axios from 'axios'
```
Comment (chansiky) [inline client/store/pathReducer.js]: duplicate axios from 'axios'

Comment (chansiky) [inline client/store/pathReducer.js]: delete this line.  this is repeated from line one.

File: client/components/Path/path-single.js [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+  }
+
+  render(){
+    const { path } = this.props
+
+    if(!path.modules) {
+      return (<h3>Please select a path</h3>)
+    }
+
+    console.log('render', this.state)
```
Comment (chansiky) [inline client/components/Path/path-single.js]: console.log - remove?

File: client/store/pathReducer.js [hunk 3]
```diff
@@ -62,20 +71,23 @@ const setSearchedPathsInCategory = (paths) => {
...
 export const getSinglePathThunk = (id) => {
   return async (dispatch) => {
-    const data = await pathsDemo
+    // const data = await pathsDemo
+    const { data } = await axios.get(`/api/paths/${id}`)
 
-    const singlepath = data.filter((item) => {
-      return item.id === id
-    })
+    console.log(data)
```
Comment (chansiky) [inline client/store/pathReducer.js]: console.log - remove?

File: script/seed.cql [hunk 4]
```diff
@@ -261,7 +261,7 @@ url: 'https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/G
 (dragonslayer)-[:PATHS]->(styled),
 (violeteyes)-[:PATHS]->(exprspath),
 (violeteyes)-[:PATHS]->(neopath)
-`).then(results => console.log(results).then(session.close())
+`).then(results => console.log(results)).then(session.close())
```
Comment (chansiky) [inline script/seed.cql]: console.log - remove? 
also dragonslayer?, violeteyes?, i will need an explanation

**Comment by jamigibbs** [comment]
@chansiky Thanks Chan! This PR is a little pre-mature because the components haven't been updated against the new routes yet (and the routes aren't completely written either).  I think I pushed this up for Justin to check out an issue I was having with a Cypher query and it ended up coming with a bunch of other stuff unfortunately. Thanks for your feedback though! I'm going to close this PR for now and submit again once I have the queries working properly.

**Pull request closed without merging**